bfapi/routes/v0.py

```diff
@@ -370,6 +376,47 @@ def _get_string(d: dict, key: str, *, nullable: bool = False, min_length: int =
         raise ValidationError('`{}` must be a string of {}–{} characters'.format(key, min_length, max_length))
     return value
 
+#
+# Imagery data
+#
+def get_search_imagery(itemType: str) -> bool:
+    log = logging.getLogger(__name__)
+    try:
+        query_params = flask.request.args
+        cloudCover = query_params.get('cloudCover')
+        PL_API_KEY = query_params.get('PL_API_KEY')
+        bbox = query_params.get('bbox')
+        acquiredDate = query_params.get('acquiredDate')
+        maxAcquiredDate = query_params.get('maxAcquiredDate')
+    except JSONDecodeError:
```

Comment:
`try` block should be removed since it does not actually raise either `JSONDecodeError` or `ValidationError`.